core/src/test/java/nl/nn/adapterframework/xslt/XsltTestBase.java

```diff
@@ -353,7 +353,7 @@ public void xpathAttrText() throws Exception {
 	@Test
 	public void xpathNodeXml() throws Exception {
 		String input = TestFileUtils.getTestFile("/Xslt/AnyXml/in.xml");
-		String expected = "<g attr=\"Euro € single quote ' double quote escaped &quot; \">Euro € single quote ' double quote \"</g>";
+		String expected = "<g attr=\"Euro € single quote ' double quote escaped &quot; newline escaped &#10;\">Euro € single quote ' double quote \"</g>";
```

Comment:
A tip I can give you for next time, try to not put too many test cases in 1 junit-test. It will make it more difficult to analyse the problem. In this case you're testing the escape part so it's understandable 👍